package/boot/uboot-envtools/files/ramips

```diff
@@ -33,6 +33,11 @@ xiaomi,miwifi-nano|\
 zbtlink,zbt-wg2626)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x10000"
 	;;
+hootoo,ht-tm05)
+        idx="$(find_mtd_index u-boot-env)"
+        [ -n "$idx" ] && \
+                ubootenv_add_uci_config "/dev/mtd$idx" "0x4000" "0x1000" "0x1000"
```

Comment:
You are sure about the position?

Reply:
See [related comments](https://forum.openwrt.org/t/hootoo-tm-05-add-lzma-loader/67687/305?u=xabolcs) on the forum!
And here is a hexdump from [arrmo's comment](https://forum.openwrt.org/t/hootoo-tm-05-add-lzma-loader/67687/306?u=xabolcs)

```
root@OpenWrt:/# cat /dev/mtd5ro | hexdump -C
00000000  00 00 00 00 31 39 32 2e  31 36 38 2e 31 2e 33 38  |....192.168.1.38|
00000010  00 00 00 00 32 35 35 2e  32 35 35 2e 32 35 35 2e  |....255.255.255.|
00000020  30 00 00 00 31 39 32 2e  31 36 38 2e 31 2e 31 00  |0...192.168.1.1.|
00000030  00 00 00 00 31 39 32 2e  31 36 38 2e 31 2e 32 30  |....192.168.1.20|
00000040  31 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |1...............|
00000050  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
00000400  00 00 00 00 50 4f 57 45  52 37 00 00 00 00 00 00  |....POWER7......|
00000410  00 00 00 00 57 69 66 69  44 69 73 6b 00 00 00 00  |....WifiDisk....|
00000420  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
00000430  00 00 00 00 31 2e 30 2e  30 2e 31 00 31 30 35 30  |....1.0.0.1.1050|
00000440  30 30 30 30 30 33 00 00  00 00 00 00 00 00 00 00  |000003..........|
00000450  00 00 00 00 00 00 00 00  00 00 00 00 30 30 3a 31  |............00:1|
00000460  63 3a 63 32 3a 30 32 3a  36 32 3a 31 63 00 00 00  |c:c2:02:62:1c...|
00000470  00 00 00 00 00 b7 bf a1  00 00 00 00 00 00 00 00  |................|
00000480  00 00 00 00 00 00 00 00  00 00 00 00 2a 00 00 00  |............*...|
00000490  32 30 31 32 30 39 30 33  6d 6b 38 36 35 31 30 78  |20120903mk86510x|
000004a0  45 43 47 45 44 53 52 50  4d 4b 4a 66 65 62 5a 59  |ECGEDSRPMKJfebZY|
000004b0  6f 6e 6c 39 37 36 32 31  47 46 00 00 00 00 00 00  |onl97621GF......|
000004c0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
00000510  40 00 00 00 39 33 64 34  38 61 39 65 65 36 63 38  |@...93d48a9ee6c8|
00000520  61 35 33 35 38 66 31 36  34 65 65 34 64 38 34 66  |a5358f164ee4d84f|
00000530  39 32 62 65 63 31 62 36  35 66 32 39 62 37 31 62  |92bec1b65f29b71b|
00000540  62 66 37 39 33 66 66 36  38 30 36 63 65 35 63 64  |bf793ff6806ce5cd|
00000550  38 39 34 35 00 00 00 00  00 00 00 00 00 00 00 00  |8945............|
00000560  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
00004000  b9 a3 e6 57 62 6f 6f 74  63 6d 64 3d 74 66 74 70  |...Wbootcmd=tftp|
00004010  00 62 6f 6f 74 64 65 6c  61 79 3d 32 00 62 61 75  |.bootdelay=2.bau|
00004020  64 72 61 74 65 3d 35 37  36 30 30 00 65 74 68 61  |drate=57600.etha|
00004030  64 64 72 3d 22 30 30 3a  41 41 3a 42 42 3a 43 43  |ddr="00:AA:BB:CC|
00004040  3a 44 44 3a 31 30 22 00  69 70 61 64 64 72 3d 31  |:DD:10".ipaddr=1|
00004050  30 2e 31 30 2e 31 30 2e  31 32 38 00 73 65 72 76  |0.10.10.128.serv|
00004060  65 72 69 70 3d 31 30 2e  31 30 2e 31 30 2e 32 35  |erip=10.10.10.25|
00004070  34 00 67 61 74 65 77 61  79 69 70 3d 31 30 2e 31  |4.gatewayip=10.1|
00004080  30 2e 31 30 2e 31 00 73  74 64 69 6e 3d 73 65 72  |0.10.1.stdin=ser|
00004090  69 61 6c 00 73 74 64 6f  75 74 3d 73 65 72 69 61  |ial.stdout=seria|
000040a0  6c 00 73 74 64 65 72 72  3d 73 65 72 69 61 6c 00  |l.stderr=serial.|
000040b0  6f 77 6e 65 72 3d 72 6d  6f 72 72 69 73 40 72 6b  |owner=rmorris@rk|
000040c0  6d 6f 72 72 69 73 2e 75  73 00 00 00 00 00 00 00  |morris.us.......|
000040d0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
00010000
```

Reply:
Right! I based this on a dump of that partition. Is it not correct?

Reply:
I think this one is correct / OK, based on the hexdump (of that partition). But by all means if I'm wrong or missing the point. Thanks!